slkarsh/color-picker-be
# Slk adjust endpoints

#### What's this PR do?

Adjusts endpoints to conform with RESTful architecture - rather than names for palettes and projects, endpoints are adjusted to utilize IDs and associated tests were updated accordingly.

#### Where should the reviewer start?

In the app.js file and then the test file

#### How should this be manually tested?

Run `npm test` to check that the tests are passing and also run requests in Postman.
